mlflow/models/evaluation/default_evaluator.py

```diff
@@ -0,0 +1,330 @@
+        artifacts = {}
+        metrics["accuracy"] = sk_metrics.accuracy_score(y, y_pred)
+        metrics["example_count"] = len(X)
+
+        # TODO: sum/mean on what data ?
+        #  [P0] Sum: Computes the (weighted) sum of the given values.
+        #  [P0] Mean: Computes the (weighted) mean of the given values.
+
+        if is_binomial:
+            if hasattr(model, "predict_proba"):
+                y_probs = model.predict_proba(X)
+                y_prob = y_probs[:, 1]
+            else:
+                y_probs = None
+                y_prob = None
+
+            confusion_matrix = sk_metrics.confusion_matrix(y, y_pred)
+            tn, fp, fn, tp = confusion_matrix.ravel()
+            metrics["true_negatives"] = tn
+            metrics["false_positives"] = fp
+            metrics["false_negatives"] = fn
+            metrics["true_positives"] = tp
+            metrics["recall"] = sk_metrics.recall_score(y, y_pred)
+            metrics["precision"] = sk_metrics.precision_score(y, y_pred)
+            metrics["f1_score"] = sk_metrics.f1_score(y, y_pred)
+
+            # TODO:
+            #  compute hinge loss, this requires calling decision_function of the model
+            #  e.g., see https://scikit-learn.org/stable/modules/generated/sklearn.svm.LinearSVC.html#sklearn.svm.LinearSVC.decision_function
+
+            if y_probs is not None:
+                metrics["roc_auc"] = sk_metrics.roc_auc_score(y, y_prob)
+                fpr, tpr, thresholds = sk_metrics.roc_curve(y, y_prob)
+                roc_curve_pandas_df = pd.DataFrame(
+                    {"fpr": fpr, "tpr": tpr, "thresholds": thresholds}
+                )
+                self._log_pandas_df_artifact(
+                    artifacts, temp_dir, roc_curve_pandas_df, run_id, "roc_curve", dataset_name
+                )
+
+                roc_auc = sk_metrics.auc(fpr, tpr)
+                metrics["precision_recall_auc"] = roc_auc
+
+                def plot_roc_curve():
+                    sk_metrics.RocCurveDisplay(
+                        fpr=fpr, tpr=tpr, roc_auc=roc_auc, estimator_name="example estimator"
+                    ).plot()
+
+                self._log_image_artifact(
+                    artifacts, temp_dir, plot_roc_curve, run_id, "roc_curve", dataset_name
+                )
+
+                def plot_lift_curve():
+                    scikitplot.metrics.plot_lift_curve(y, y_probs)
+
+                self._log_image_artifact(
+                    artifacts, temp_dir, plot_lift_curve, run_id, "lift_curve", dataset_name
+                )
+
+            def plot_confusion_matrix():
+                sk_metrics.ConfusionMatrixDisplay.from_predictions(y, y_pred)
+
+            self._log_image_artifact(
+                artifacts, temp_dir, plot_confusion_matrix, run_id, "confusion_matrix", dataset_name
+            )
+
+        self._log_metrics(run_id, metrics, dataset_name)
+
+        if evaluator_config.get('log_model_explainality', True):
```

Comment:
```suggestion
        if evaluator_config.get('log_model_explainability', True):
```

Reply:
@WeichenXu123 Can we document the configs accepted by the default evaluator, as well as the default value for each config, in the docstring for `DefaultEvaluator.evaluate()` *and* the docstring for `mlflow.evaluate()`?